app/lib/cfe_connection.rb

```diff
@@ -14,7 +14,7 @@ def status
 
   def state_benefit_types
     response = cfe_connection.get("state_benefit_type")
-    response.body.map { _1["name"] }.uniq
+    response.body.map { |benefit| [benefit["name"], benefit["exclude_from_gross_income"]] }.uniq
```

Comment:
This is a bit subjective, but I don't like having `state_benefit_types` return an array of arrays containing a bunch of booleans with no indication (unless you read the code inside the method) of what those booleans mean. I'd prefer it to return an array of hashes where each hash has a `name` and `exclude_from_gross_income` key. (At which point you could just return the whole body, since the body already has exactly those things.

Reply:
(Feel free to ignore this one if you disagree - like I say, this is subjective)

Reply:
I agree. Makes sense that cfe_connection is doing as little as possible to the data so I have set it to return everything